pkgs/by-name/fl/flake-checker/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "flake-checker";
+  version = "0.1.16";
+
+  src = fetchFromGitHub {
+    owner = "DeterminateSystems";
+    repo = pname;
```

```suggestion
    repo = "flake-checker";
```
allow to overwrite pname.